_posts/2019-05-24-varying-slopes/varying_slopes.Rmd

```diff
@@ -0,0 +1,748 @@
+ 5 sigma         0.473 0.0264  0.420  0.523  424.    1.00
+ 6 sigma_cafe_1  0.967 0.163   0.694  1.29    97.9   1.00
+ 7 sigma_cafe_2  0.607 0.129   0.386  0.861   42.3   1.03
+ 8 b            -1.14  0.141  -1.43  -0.864   95.1   1.00
+ 9 a             3.66  0.218   3.22   4.07    75.3   1.01
+10 a_cafe_1      4.20  0.192   3.83   4.57    83.9   1.01
+11 b_cafe_1     -1.13  0.251  -1.63  -0.664   63.6   1.02
+12 a_cafe_2      2.17  0.195   1.79   2.54    59.3   1.01
+13 b_cafe_2     -0.923 0.260  -1.42  -0.388   46.0   1.01
+14 a_cafe_3      4.40  0.195   4.02   4.79    56.7   1.01
+15 b_cafe_3     -1.97  0.258  -2.52  -1.51    43.9   1.01
+16 a_cafe_4      3.22  0.199   2.80   3.57    58.7   1.02
+17 b_cafe_4     -1.20  0.254  -1.70  -0.713   36.3   1.01
+18 a_cafe_5      1.86  0.197   1.45   2.20    52.8   1.03
+19 b_cafe_5     -0.113 0.263  -0.615  0.390   34.6   1.04
+20 a_cafe_6      4.26  0.210   3.87   4.67    43.4   1.02
+21 b_cafe_6     -1.30  0.277  -1.80  -0.713   41.4   1.05
+22 a_cafe_7      3.61  0.198   3.23   3.98    44.9   1.01
+23 b_cafe_7     -1.02  0.263  -1.51  -0.489   37.7   1.03
+24 a_cafe_8      3.95  0.189   3.59   4.31    73.1   1.01
+25 b_cafe_8     -1.64  0.248  -2.10  -1.13    60.7   1.02
+26 a_cafe_9      3.98  0.212   3.57   4.37    76.3   1.03
+27 b_cafe_9     -1.29  0.273  -1.83  -0.776   57.8   1.05
+28 a_cafe_10     3.60  0.187   3.24   3.96   104.    1.01
+29 b_cafe_10    -1.00  0.245  -1.47  -0.512   70.4   1.00
+30 a_cafe_11     1.95  0.200   1.56   2.35    55.9   1.03
+31 b_cafe_11    -0.449 0.266  -1.00   0.0619  42.5   1.04
+32 a_cafe_12     3.84  0.195   3.46   4.22    76.0   1.02
+33 b_cafe_12    -1.17  0.259  -1.65  -0.670   62.5   1.03
+34 a_cafe_13     3.88  0.201   3.50   4.29    62.2   1.02
+35 b_cafe_13    -1.81  0.270  -2.30  -1.29    48.3   1.03
+36 a_cafe_14     3.19  0.212   2.82   3.61    65.9   1.07
+37 b_cafe_14    -0.961 0.278  -1.49  -0.401   49.9   1.06
+38 a_cafe_15     4.46  0.212   4.08   4.91    62.0   1.09
+39 b_cafe_15    -2.20  0.290  -2.72  -1.59    47.8   1.11
+40 a_cafe_16     3.41  0.193   3.02   3.78    62.7   1.02
+41 b_cafe_16    -1.07  0.253  -1.54  -0.567   48.5   1.05
+42 a_cafe_17     4.22  0.201   3.82   4.60    58.7   1.01
+43 b_cafe_17    -1.24  0.273  -1.74  -0.703   43.8   1.01
+44 a_cafe_18     5.77  0.210   5.34   6.18    66.0   1.02
+45 b_cafe_18    -1.05  0.284  -1.61  -0.511   49.8   1.02
+46 a_cafe_19     3.23  0.203   2.88   3.65    52.7   1.02
+47 b_cafe_19    -0.232 0.276  -0.808  0.243   45.2   1.01
+48 a_cafe_20     3.74  0.212   3.35   4.21    48.2   1.04
+49 b_cafe_20    -1.09  0.281  -1.58  -0.506   36.5   1.05
+```
+
+So what do we have? If you run this "live", for the rows `a_cafe_n` resp. `b_cafe_n`, you see a nice alternation of white and red coloring: For all cafés, the inferred slopes are negative.
+The inferred slope prior (`b`) is around -1.14, which is not too far off from the value we used for sampling: 1.
```

Comment:
```suggestion

The inferred slope prior (`b`) is around -1.14, which is not too far off from the value we used for sampling: 1.
```

Reply:
sorry I fail to see the difference?

Reply:
I added a blank line above it, since the other paragraphs were separated that way.

Reply:
ah I see, thanks!